modules/actions.js

```diff
@@ -1,33 +1,28 @@
-class Data {
-  static getLocalStorage() {
-    let allScores;
-    if (localStorage.getItem('allScores') === null) {
-      allScores = [];
-    } else {
-      allScores = JSON.parse(localStorage.getItem('allScores'));
-    }
-    return allScores;
-  }
-}
+const api = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/:Fh0ZLhL9PMX1g7EtaJDa/scores';
 
+/* eslint-disable  quote-props */
```

Comment:
Please don't disable the linters without having big issues:
![image](https://user-images.githubusercontent.com/42799579/202021054-40223a9b-5072-4415-a03c-14e731cd3656.png)

In this case, you can just remove the quotes from your properties:
![image](https://user-images.githubusercontent.com/42799579/202021155-d9bca4e0-4b8f-47d9-bdef-c07ebd2a252c.png)

If this is causing a breaking bug or anything hard to solve, kindly consider adding the eslint-disable again. Please keep in mind that linters are made to standardize/help with minor errors inside your code 😉.

Reply:
@Gopxfs Hi. I tried just removing the quotes but then it was given me an error about missing the quotes, so I added the rule to disable it again. But I will try to fix it again with another method.

Reply:
> @Gopxfs Hi. I tried just removing the quotes but then it was given me an error about missing the quotes, so I added the rule to disable it again. But I will try to fix it again with another method.

Hello, try to use `user` and `score` with any quotes at all, I tried this and it working 😉.

Reply:
This was my result:
```JavaScript
{
  user: name,
  score,
}
```

Reply:
I will try that. Thank you 👍